Domain.Authentication/Queries/GenerateBearerToken/GenerateBearerTokenQuery.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using Diagnosea.Submarine.Domain.User.Enums;
+using MediatR;
+
+namespace Diagnosea.Submarine.Domain.Authentication.Queries.GenerateBearerToken
+{
+    public class GenerateBearerTokenQuery : IRequest<string>
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
```

Comment:
Consider making these more inline with what a bearer token is actually made up of. What we use them for is an usage detail and this implementation should be dumb.